qa/tasks/cephfs/kernel_mount.py

```diff
@@ -92,14 +94,15 @@ def umount(self, force=False):
             raise e
 
         self.mounted = False
-        self.cleanup_netns()
```

Comment:
BTW, you removed all the cleanup_netns() in fuse_mount.py and kernel_mount.py, then when and where will this be called ? Or did you forget to update in cleanup() ?

Reply:
Yeah, looks like I failed to notice that a hunk was rejected from my patch.

Reply:
Fixed. Thanks!